dev/releases/3.0.0-rc.1.md

```diff
@@ -66,15 +66,15 @@ On May 10, 2019, TiDB 3.0.0-rc.1 is released. The corresponding TiDB-Ansible ver
     - Fix the issue of failing to elect Leader by PreVote
     - Support fast dropping the “propose” and “read” requests that are to fail to avoid blocking the subsequent requests
     - Fix the deadlock issue of Lease
-- Fix the issue that a hot store makes incorrect statistics of keys [#1487](https://github.com/pingcap/pd/pull/1487)
-- Support forcibly rebuilding a PD cluster from a single PD node [#1485](https://github.com/pingcap/pd/pull/1485)
-- Fix the issue that `regionScatterer` might generate an invalid `OperatorStep` [#1482](https://github.com/pingcap/pd/pull/1482)
-- Fix the too short timeout issue of the `MergeRegion` operator [#1495](https://github.com/pingcap/pd/pull/1495)
-- Support giving high priority to hot region scheduling [#1492](https://github.com/pingcap/pd/pull/1492)
-- Add the metrics for recording the time of handling TSO requests on the PD server side [#1502](https://github.com/pingcap/pd/pull/1502)
-- Add the corresponding Store ID and Address to the metrics related to the store [#1506](https://github.com/pingcap/pd/pull/1506)
-- Support the `GetOperator` service [#1477](https://github.com/pingcap/pd/pull/1477)
-- Fix the issue that the error cannot be sent in the Heartbeat stream because the store cannot be found [#1521](https://github.com/pingcap/pd/pull/1521)
++ Fix the issue that a hot store makes incorrect statistics of keys [#1487](https://github.com/pingcap/pd/pull/1487)
++ Support forcibly rebuilding a PD cluster from a single PD node [#1485](https://github.com/pingcap/pd/pull/1485)
++ Fix the issue that `regionScatterer` might generate an invalid `OperatorStep` [#1482](https://github.com/pingcap/pd/pull/1482)
++ Fix the too short timeout issue of the `MergeRegion` operator [#1495](https://github.com/pingcap/pd/pull/1495)
++ Support giving high priority to hot region scheduling [#1492](https://github.com/pingcap/pd/pull/1492)
++ Add the metrics for recording the time of handling TSO requests on the PD server side [#1502](https://github.com/pingcap/pd/pull/1502)
++ Add the corresponding Store ID and Address to the metrics related to the store [#1506](https://github.com/pingcap/pd/pull/1506)
++ Support the `GetOperator` service [#1477](https://github.com/pingcap/pd/pull/1477)
++ Fix the issue that the error cannot be sent in the Heartbeat stream because the store cannot be found [#1521](https://github.com/pingcap/pd/pull/1521)
 
 ## TiKV
 
```

Comment:
**Modify line 86 to:**
- Support `block cache` sharing among different `column families`